src/utils.js

```diff
@@ -588,4 +588,12 @@ export function isSrcdocSupported(doc) {
 
 export function cloneJson(obj) {
   return JSON.parse(JSON.stringify(obj));
+}
+
+export function inIframe() {
```

Comment:
Hmm; I would hate to break backwards compatibility.  Suren and I will brainstorm some techniques to see if we can come up with a backwards compatible solution that still solves this problem, and if so we'll update the pull-request.  Thanks!